setup.py

```diff
@@ -101,9 +97,7 @@ def run(self):
     cmdclass={
         'install': PostInstallCommand,
     },
-    install_requires=SETUP_REQUIRES + INSTALL_REQUIRES,
-    setup_requires=SETUP_REQUIRES,
-    tests_require=INSTALL_REQUIRES + SETUP_REQUIRES + TEST_REQUIRES,
+    install_requires=INSTALL_REQUIRES,
     packages=find_packages(exclude=[
         '*.tests', '*.tests.*', 'tests.*', 'tests']),
     include_package_data=True,
```

Comment:
Below, please remove forseti_scanner and forseti_notifier.

Reply:
done!